recipes/forestci/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "forestci" %}
+{% set version = "0.1" %}
+{% set sha256 = "e706ec046c802c2e1036fd8ee306f1d9841b1a9239d0346f8c1849c5966578e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - scikit-learn
+    - numpy
+    - nose
+
+test:
+  imports:
+    - forestci
+ 
+about:
+  home: https://github.com/scikit-learn-contrib/forest-confidence-interval
+  license: MIT
+  license_file: LICENSE.txt
+  summary: 'calculate confidence intervals for scikit-learn random forest regression or classification objects'
+
+  description: |
+    a Python module for calculating variance and adding confidence intervals to scikit-learn
+    random forest regression or classification objects. The core functions calculate an in-bag and 
+    error bars for random forest objects
+  doc_url: http://contrib.scikit-learn.org/forest-confidence-interval/
+  dev_url: https://github.com/scikit-learn-contrib/forest-confidence-interval
+
+extra:
+  recipe-maintainers:
```

Comment:
Even though the conda-forge linter said `recipe-maintainers` block was ok, I'm not sure if the build tools actually support the form `realname (github id)`. Typically I've only seen the github id, although it would be nice to all for the maintainer's full name (I can appreciate this since for historical reasons, I have a pretty obscure github id myself)